src/documentation_glossary.jl

```diff
@@ -231,25 +242,39 @@ define!(
 #
 #
 # Problems
+_problem(args...; kwargs...) = glossary(:Problem, args...; kwargs...)
+
 define!(
     :Problem,
     :Constrained,
     (; M="M", p="p") -> """
     ```math
 \\begin{aligned}
-\\min_{$p ∈ $(_tex(:Cal, M))} & f($p)\\\\
+$(_tex(:argmin))_{$p ∈ $(_math(:M; M=M))} & f($p)\\\\
 $(_tex(:text, "subject to"))$(_tex(:quad))&g_i($p) ≤ 0 \\quad $(_tex(:text, " for ")) i= 1, …, m,\\\\
 \\quad & h_j($p)=0 \\quad $(_tex(:text, " for ")) j=1,…,n,
 \\end{aligned}
 ```
 """,
 )
+define!(:Problem, :Default, (; M="M", p="p") -> """
+                        ```math
+                        $(_tex(:argmin))_{$p ∈ $(_math(:M; M=M))} f($p)
+                        ```
+                        """)
 define!(
     :Problem,
-    :Default,
-    (; M="M", p="p") -> "\n```math\n$(_tex(:argmin))_{$p ∈ $(_tex(:Cal, M))} f($p)\n```\n",
+    :NonLinearLeastSquares,
+    (; M="M", p="p") -> """
+```math
+$(_tex(:argmin))_{$p ∈ $(_math(:M; M=M))} $(_tex(:frac,1,2)) $(_tex(:sum))_{i=1}^n $(_tex(:abs, "f_i($p)"))^2
+```
+
+where ``f: $(_math(:M; M=M)) → ℝ^n`` is written with component functions ``f_i: $(_math(:M; M=M)) → ℝ``,
```

Comment:
The number of components in an NLS problem is denoted by `m` in most places I think?

Reply:
Oh then I have to check again, yes I confused that to be `n` for some time but had thought I fixed all occurrences.
Letters. Nearly as complicated as Numbers.